src/components/Card/Card.tsx

```diff
@@ -8,10 +8,12 @@ interface PropsCard {
 
 const Card: React.FC<PropsCard> = ({ title, image }) => {
```

Comment:
Should we remove the React.FC from our functional components now and follow [this example](https://github.com/typescript-cheatsheets/react/blob/main/README.md#function-components)?

Reply:
Just reformatted the Form component to not include React.FC and received the following error from eslint : 
"Line 10:14:  Missing return type on function  @typescript-eslint/explicit-module-boundary-types"
This was not a warning we got before, do we want to ignore this eslint warning and/or uninstall?

Reply:
Would adding the return type of JSX.Element fix that error?